### PR TITLE
Eo controller edge cases and error handling

### PR DESCRIPTION
**Pull request by KMPWeiland**
Type of change

  - [x] New feature
  - [ ] Bug Fix

Check the correct boxes

  - [x] This code broke nothing
  - [ ] This code broke some stuff
  - [ ] This code broke everything

Implements/Fixes:

  - Description of work...
  - Add basic error handling to Executive Orders controller.
  -Add edge case and error handling tests to the Executive Orders spec.

Testing Changes:

  - [x] I have fully tested my code 
  - [x] All tests are passing
  - [ ] Some tests are failing
  - [ ] All tests are failing

  - [ ] No previous tests have been changed
  - [ ] Some previous tests have been changed
  - [ ] All of the previous tests have been changed (Please describe what in the world happened that all of the previous tests needed changing.)

Checklist:

  - [x] I have reviewed my code
  - [x] The code will run locally
  - [ ] My code has no unused/commented out code
  - [ ] I have commented my code, particularly in hard-to-understand areas

(Optional) What questions do you have? Anything specific you want feedback on?

  * 

(For Fun!) Please include a link to a gif [or add an emoji] of your feelings about this branch.

Link:

### CODE REVIEW

File: spec/requests/api/v1/executive_orders_request_spec.rb [hunk 1]
```diff
@@ -58,4 +58,95 @@
       end
     end
   end
+
+  describe "Edge Cases and Error Handling" do
```
Comment (elysableu) [inline spec/requests/api/v1/executive_orders_request_spec.rb:62]: Thank you for writing all these tests!  You rock!

**Review by elysableu — APPROVED** [review]
You're a testing super star!
